guide/designing-products/user-research.md

```diff
@@ -98,7 +98,7 @@ Some of the following links have been around for a long time and provide a huge
 
 ### An unknowable userbase
 
-The Bitcoin protocol works with addresses and has otherwise no sense of identity that allows us to find out who uses the system and performs various functions. Analyzing the blockchain allows for some basic transaction behavior analysis, but analysis cannot tie it to specific persons or locations. While this is generally a fantastic user benefit for privacy and security, it is harder to understand user behavior and recruit users for research.
+The Bitcoin protocol only knows its users through pseudonymous addresses. We can analyze the Bitcoin blockchain for fundamental transaction behavior analysis, but further analysis of addresses and transactions can be complicated. While this provides incredible privacy and security for users, it can make things challenging for user researchers.
```

Comment:
```suggestion
Bitcoin only knows its users through pseudonymous addresses. We can analyze on-chain metrics for things like fundamental transaction behaviors, but analysis of addresses and transactions beyond this can be complicated. While Bitcoin's privacy provides is a fundamental aspect, it can make things challenging for user researchers.
```